circle.yml

```diff
@@ -11,7 +11,11 @@ test:
     - ./scripts/iOS.sh
   post:
     - carthage build --no-skip-current && test -d Carthage/Build/iOS/Swiftilities.framework || exit 1
-    - bundle exec pod lib lint
+    # This is to workaround the fact that cocoapods wants to set up the master cocoapods on pod lib lint
```

Comment:
My updated version, adapted from what I'm going to use on BonMot, in case you want it:

```yaml
# This is to work around the fact that CocoaPods wants to set up the master CocoaPods specs
# repo on pod lib lint, which takes several minutes on CircleCI. Without dependancies, this
# isn't a nescessary step. We can point it at any other (smaller) Git repo to speed up this
# step significantly. Because recursion is fun, let's point it at ourselves.
- bundle exec pod lib lint --sources="https://github.com/Raizlabs/Swiftilities"
```